### PR TITLE
feat: operation-packaging-announcement

### PR DESCRIPTION
**Pull request by teolemon**
### What
- feat: operation-packaging-announcement

### CODE REVIEW

File: lang/en/texts/operation-packaging-announcement.html [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+<!-- wp:paragraph {"style":{"color":{"text":"#008c8c"}},"fontSize":"medium"} -->
+<p class="has-text-color has-medium-font-size" style="color:#008c8c">Your mission "What's around my yogurt" is now complete! WELL DONE. </p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"style":{"color":{"text":"#008c8c"}},"fontSize":"medium"} -->
+<p class="has-text-color has-medium-font-size" style="color:#008c8c">You are an expert eater! 😊 </p>
+<!-- /wp:paragraph -->
+
+<!-- wp:image {"id":2918,"width":596,"height":375,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="https://blog.openfoodfacts.org/wp-content/uploads/2022/12/Other-visual-packaging-1 .png" alt="" class="wp-image-2918" width="596" height="375"/></figure>
```
Comment (alexgarel) [inline lang/en/texts/operation-packaging-announcement.html:416]: This image does not exists. (even if I remove the space)

**Review by alexgarel — APPROVED** [review]
LGTM appart the last image.

What is the intent of putting it here ? It's to trigger translations ?

File: lang/en/texts/operation-packaging-announcement.html (hunk 1 above)
Comment (teolemon) [inline lang/en/texts/operation-packaging-announcement.html:416]:
```suggestion
<figure class="wp-block-image size-full is-resized"><img src="https://blog.openfoodfacts.org/wp-content/uploads/2022/12/Autres-visuels-emballages-1.png" alt="" class="wp-image-2918" width="596" height="375"/></figure>
```

File: lang/en/texts/operation-packaging-announcement.html [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+<!-- wp:paragraph -->
+<p></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>The <strong>citizen mobilization</strong> will begin on January 12 following the launch event (we tell you everything below)!&nbsp;</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>🤳🥫 To participate in this one-month operation, you will only need your smartphone (or computer) with <a href="https://fr.openfoodfacts.org/application-mobile-open-food- facts">the Open Food Facts app</a> and your groceries! 🃏If you have an extra kitchen scale, you hold the joker to provide us with another key piece of data - the weight of the empty packaging.&nbsp;</p>
```
Comment (teolemon) [inline lang/en/texts/operation-packaging-announcement.html:36]:
```suggestion
<p>🤳🥫 To participate in this one-month operation, you will only need your smartphone (or computer) with <a href="https://world.openfoodfacts.org/open-food-facts-mobile-app">the Open Food Facts app</a> and your groceries! 🃏If you have an extra kitchen scale, you hold the joker to provide us with another key piece of data - the weight of the empty packaging.&nbsp;</p>
```

**Comment by teolemon** [comment]
@alexgarel I'm closing this and moving it to https://github.com/openfoodfacts/openfoodfacts-translations/pull/83

**Pull request closed without merging**